app/routes/__frontend/resources/index.mdx

```diff
@@ -1,23 +1,35 @@
 ---
 meta:
-  title: 'Virtual Coffee Member Resources'
-  description: 'A collection of resources for Virtual Coffee members'
+  title: Virtual Coffee Resources
+  description: A collection of resources we provide for our members and all developers.
 hero:
   Hero: UndrawFolder
-  heroHeader: Member Resources
+  heroHeader: Virtual Coffee Resources
 tags:
   - memberresources
   - topnav
 order: 1
 ---
 
+import LeadText from '~/components/content/LeadText';
 import TextContainer from '~/components/content/TextContainer';
 import FileIndex from '~/components/content/FileIndex';
+import { Link } from '@remix-run/react';
 
 <TextContainer background="light" showBackToTopLink={false}>
 
+<LeadText>
+
+We compile and provide guides and resources for Virtual Coffee's members as community documentation in <Link to="/resources/virtual-coffee-handbook">Virtual Coffee Handbook</Link>. And we also have resources for all developers that live in <Link to="/resources/developer-resources">Developer Resources</Link>. Feel free to check them out!
```

Comment:
We compile and provide guides and resources for Virtual Coffee's members as community documentation in the <Link to="/resources/virtual-coffee-handbook">Virtual Coffee Handbook</Link>. We also have resources for all developers that live in <Link to="/resources/developer-resources">Developer Resources</Link>. Feel free to check them out!